### PR TITLE
supported modules on AA

### PR DESCRIPTION
**Pull request by AlonMagrafta**

### CODE REVIEW

File: content/rs/release-notes/rs-6-2-18.md [hunk 1]
```diff
@@ -34,6 +35,10 @@ The following table shows the MD5 checksums for the available packages:
 
 ## Features and enhancements
 
+## General Availability of Active-Active databases with RedisJSON
+
+The capability of indexing, query and full-text search of nested JSON documents are now also GA on Active-Active (achieved by the combination of RedisJSON with RediSearch).
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-2-18.md:40]:
```suggestion
Index, query, and full-text search of nested JSON documents are now available on Active-Active  databases by using the combination of RedisJSON with RediSearch.
```

File: content/rs/databases/active-active/planning.md [hunk 2]
```diff
@@ -69,10 +69,10 @@ See [Synchronizing cluster node clocks]({{<relref "/rs/clusters/configure/sync-c
 
 ## Redis modules {#redis-modules}
 
-Not all Redis modules are compatible with Active-Active databases. Below is a list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
-
-- [RediSearch 2.x in Redis Enterprise Software 6.0 and higher]({{< relref "/modules/redisearch/redisearch-active-active.md" >}})
-- RedisGears
+Several Redis modules are compatible with Active-Active databases. Find the list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
+{{< note >}}
+Starting v6.2.18 the capability of indexing, query and full-text search of nested JSON documents are now also GA on Active-Active (achieved by the combination of RedisJSON with RediSearch).
```
Comment (kaitlynmichael) [inline content/rs/databases/active-active/planning.md:74]:
```suggestion
Starting v6.2.18 the capability of index, query, and full-text search of nested JSON documents, are now also available for Active-Active databases by using a combination of RedisJSON with RediSearch.
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]
I reworded a bit for clarity. Once you approve those changes I can push this out.

File: content/rs/databases/active-active/planning.md [hunk 3]
```diff
@@ -69,10 +69,10 @@ See [Synchronizing cluster node clocks]({{<relref "/rs/clusters/configure/sync-c
 
 ## Redis modules {#redis-modules}
 
-Not all Redis modules are compatible with Active-Active databases. Below is a list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
-
-- [RediSearch 2.x in Redis Enterprise Software 6.0 and higher]({{< relref "/modules/redisearch/redisearch-active-active.md" >}})
-- RedisGears
+Several Redis modules are compatible with Active-Active databases. Find the list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
+{{< note >}}
+Starting v6.2.18 the capability of index, query, and full-text search of nested JSON documents, are now also available for Active-Active databases by using a combination of RedisJSON with RediSearch.
```
Comment (lanceleonard) [inline content/rs/databases/active-active/planning.md:74]:
```suggestion
Starting with v6.2.18, you can now index, query, and perform full-text searches of nested JSON documents in Active-Activate databases, by combining RedisJSON and RediSearch.
```

File: content/rs/release-notes/rs-6-2-18.md [hunk 4]
```diff
@@ -34,6 +35,10 @@ The following table shows the MD5 checksums for the available packages:
 
 ## Features and enhancements
 
+## General Availability of Active-Active databases with RedisJSON
+
+Index, query, and full-text search of nested JSON documents are now available on Active-Active  databases by using the combination of RedisJSON with RediSearch.
```
Comment (lanceleonard) [inline content/rs/release-notes/rs-6-2-18.md:40]:
```suggestion
Active-Active databases now support Index, query, and full-text search of nested JSON documents when combining RedisJSON with RediSearch.
```

**Comment by AlonMagrafta** [comment]
Thanks @kaitlynmichael

File: content/rs/release-notes/rs-6-2-18.md [hunk 5]
```diff
@@ -45,6 +50,7 @@ When enabled, private key encryption encrypts private keys stored in the cluster
 ### Active-Active support for MEMORY USAGE command
 
 Redis Enterprise Active-Active databases now support the [MEMORY USAGE](https://redis.io/commands/memory-usage/) command, which simplifies troubleshooting and lets applications detect anomalous behavior and dangerous trends.
+Find out more about [`Active-Active with RedisJSON`]({{<relref "//modules/redisjson/active-active">}})
```
Comment (lanceleonard) [inline content/rs/release-notes/rs-6-2-18.md:53]:
```suggestion
To learn more, see  [`Active-Active with RedisJSON`]({{<relref "//modules/redisjson/active-active">}}).
```

**Review by lanceleonard — APPROVED** [review]

File: content/rs/databases/active-active/planning.md [hunk 6]
```diff
@@ -69,10 +69,10 @@ See [Synchronizing cluster node clocks]({{<relref "/rs/clusters/configure/sync-c
 
 ## Redis modules {#redis-modules}
 
-Not all Redis modules are compatible with Active-Active databases. Below is a list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
-
-- [RediSearch 2.x in Redis Enterprise Software 6.0 and higher]({{< relref "/modules/redisearch/redisearch-active-active.md" >}})
-- RedisGears
+Several Redis modules are compatible with Active-Active databases. Find the list of [compatible Redis modules]({{< relref "/modules/enterprise-capabilities.md" >}}).
+{{< note >}}
+Starting with v6.2.18, you can now index, query, and perform full-text searches of nested JSON documents in Active-Activate databases, by combining RedisJSON and RediSearch.
```
Comment (rrelledge) [inline content/rs/databases/active-active/planning.md:74]: Noticed a typo.
```suggestion
Starting with v6.2.18, you can index, query, and perform full-text searches of nested JSON documents in Active-Active databases by combining RedisJSON and RediSearch.
```

**Comment by AlonMagrafta** [comment]
@kaitlynmichael @lanceleonard can we push? I see there is still change request by Kaitlyn - do we have open items?

**Pull request merged**